packages/api-derive/src/tx/signingInfo.ts

```diff
@@ -63,6 +66,19 @@ function signingHeader (api: DeriveApi): Observable<Header> {
   );
 }
 
+interface Aura {
+  slotDuration: u64 & AugmentedConsts<'rxjs'>;
+}
+
+function babeOrAuraPeriod (api: DeriveApi): BN | undefined {
+  const period = api.consts.babe?.expectedBlockTime ||
+    // this will be present ones https://github.com/paritytech/polkadot-sdk/pull/3732 is merged
+    (api.consts['aura'] as unknown as Aura)?.slotDuration ||
+    api.consts.timestamp?.minimumPeriod;
```

Comment:
Hi everyone, I notice that we do `x2` (`.muln(2)`) for the `minimumPeriod` previously, but now we removed it here. I didn't see we mentioned this change anywhere, any reason for this? I just want to confirm that this is intentional. Thank you!

Reply:
Hey @sinzii, thanks for the heads up! This was an error, this [pr](https://github.com/polkadot-js/api/pull/5829) fix it.
Thx!

Reply:
Awesome, thanks for the quick fix!